packages/core/src/destination-kit/types.ts

```diff
@@ -23,6 +23,8 @@ export interface ExecuteInput<Settings, Payload> {
   page?: string
   /** The data needed in OAuth requests */
   readonly auth?: AuthTokens
+  /** The Flagon features available in the request */
```

Comment:
```suggestion
  /** The features available in the request based on customer workspace */
```

I think including word Flagon may confuse people.